### DISCUSSION TITLE
Does anyone has tried to integrate `Cloudinary` with `Gatsby-plugin-image`

### DISCUSSION THREAD

**Discussion by rifqirosyidi** [gatsbyjs/gatsby#30611] [Help]
Hi,

just want to ask, is it possible to integrate **cloudinary** with **gatsby-plugin-image**, While I’ve found some information on cloudinary + gatsby-image, I haven’t seen any article/docs covering cloudinary with gatsby-plugin-image.

**Comment by ascorbic** [comment]
Hi. This needs somebody to update the Cloudinary plugin. I have some sample code though, which needs updating to [the latest version of the API](https://www.gatsbyjs.com/docs/how-to/plugins-and-themes/adding-gatsby-image-support/#adding-a-custom-image-component), but could serve as a starting point: https://github.com/ascorbic/image-utils-demo/blob/main/src/use-cloudinary-image.ts

There's [a bit of discussion](https://github.com/gatsbyjs/gatsby/discussions/28241#discussioncomment-138599) about it. Maybe @tpiros has thoughts.

**Reply by tpiros** [reply]
Also adding @Chuloo to the thread. I will take a look at this soon :)

**Reply by tpiros** [reply]
@rifqirosyidi - depending on your use-case, you may also look at https://www.gatsbyjs.com/plugins/gatsby-transformer-cloudinary/ which does use Cloudinary and renders Gatsby Image nodes (with the fluid and fixed props) that you can use in your app. If you get a chance, please let us know your use-case(s) and I'm sure we can figure something out together :)

**Reply by rifqirosyidi** [reply]
I already tried using transformer-cloudinary it works fine. but from what i know it still using gatsby-image instead of gatsby-plugin-image. just want to keep up to date with gatsby-plugin-image if it is possible.
